data-processing/entities/citations/commands/upload_citations.py

```diff
@@ -34,9 +36,60 @@ def get_description() -> str:
     def get_arxiv_ids_dirkey(self) -> str:
         return "citations-locations"
 
+    def _get_bibitem_texts_from_bibitems(
+        self,
+        arxiv_id: ArxivId,
+        bibitems: List[Bibitem]
+    ) -> Dict[str, str]:
+        def acceptable_str(maybe_str: Optional[str]) -> bool:
```

Comment:
can document in here a couple of example arXiv IDs on which this happens?

Reply:
A couple examples of where this would return False?
If so, I don't know of any examples. I'm mostly going off the signature for the extractor's [_extract_key()](https://github.com/allenai/scholarphi/blob/d614c3a73dcc97946f3885a336ab0795512ed9ea/data-processing/entities/citations/extractor.py#L43) (which says it returns an Optional[str]), and a kind of theoretical idea of what wouldn't be useful at all (empty strings for either the bib item key or text). I can add a comment explaining that.